pkgs/by-name/ck/ckb/package.nix

```
@@ -0,0 +1,56 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  ckb,
+  openssl,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+  versionCheckHook,
+  testers,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "ckb";
+  version = "0.202.0";
+
+  src = fetchFromGitHub {
+    owner = "nervosnetwork";
+    repo = "ckb";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-LE+w5RHkDk+1Zu2RZgPyHbHJ92C1oAxGEcVc3Qo2lQc=";
+  };
+
+  cargoHash = "sha256-R7XV//e6n6afbad7HgwlcNwQRdO0xFZ0UGy1SU294Fs=";
+
+  buildFeatures = lib.optionals (stdenv.hostPlatform.isDarwin) [ "portable" ];
+
+  nativeBuildInputs = [
+    rustPlatform.bindgenHook
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ];
+
+  env.OPENSSL_NO_VENDOR = true;
+  __darwinAllowLocalNetworking = true;
+
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [ versionCheckHook ];
+
+  meta = {
+    description = "The Nervos CKB is a public permissionless blockchain, and the layer 1 of Nervos network.";
```

This is minor, but blocking:

```suggestion
    description = "Public permissionless blockchain, and the layer 1 of Nervos network";
```

See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

> - Not start with the definite or an indefinite article.
> - Not start with the package name.
>   - More generally, it should not refer to the package name.
> - Not end with a period (or any punctuation for that matter).

In this case "the" is a definite article.

(Note, if needed you can include more detail and/or examples in `meta.longDescription`)